data/json/vehicles/trains.json

```diff
@@ -124,5 +124,129 @@
       { "x": 8, "y": 2, "parts": [ "frame_horizontal", "board_horizontal", "roof" ] },
       { "x": 8, "y": 3, "parts": [ "frame_ne", "board_ne", "roof" ] }
     ]
+  },
+  {
+    "id": "motorized_draisine_2seats",
+    "type": "vehicle",
+    "name": "Two-Seated Motorized Draisine",
+    "blueprint": [
+      [ "O-O" ],
+      [ "|#|" ],
+      [ "|X|" ],
+      [ "|#|" ],
+      [ "O-O" ]
+    ],
+    "parts": [
+      { "x": 0, "y": 1, "parts": [ "xlframe_vertical_2", { "part": "tank_small", "fuel": "gasoline" } ] },
```

Comment:
Please use parts for the the rest of these entries:
```
{ "x": 0, "y": 0, "parts": [ "seat", "controls" ] },
{ "x": 0, "y": 1, "parts": [ "engine_vtwin", "alternator_motorbike", "battery_motorbike" }
```
it's up to you how much you want to compact them but there's no need to have a separate entry for each part.

Reply:
eh i've used vehicleDef.py script to make templates, it produces lines like that